### PR TITLE
chore(docs): link docs to Pocket codebase, YouTube

### PR DESCRIPTION
**Pull request by danieldreier**
Am I missing any YouTube videos you all are aware of?

### CODE REVIEW

**Review by skorfmann — APPROVED** [review]
There are likely more, but I think it's sufficient 👍

File: website/docs/cdktf/examples.html.md [hunk 1]
```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
...
-TODO Add information here with a link.
+* Watch the CDK for Terraform engineering team build a simple end to end serverless application in an [end to end livestream](https://www.youtube.com/watch?v=Ey0SW0c6p8c)
+* [Getting Started with CDK for Terraform and Python](https://www.youtube.com/watch?v=Ee2qh-pEC5k&t=258s), by Charles mcLaughlin of ShopStyle
+* [Extending constructs of the CDK for Terraform](https://www.youtube.com/watch?v=cfU-WOGdNqA)
 
-## Pocket public codebase?
+### Release demos
+[0.4: Go support, Asset construct, Terraform Cloud integration](https://www.youtube.com/watch?v=TTfFAIeSqgo). Includes a [GKE demo](https://youtu.be/TTfFAIeSqgo?t=1573).
+[0.5: Convert and watch command demos](https://www.youtube.com/watch?v=4caW8WJM4h4&t=1s)
+[0.6: snapshot testing](https://www.youtube.com/watch?v=9Is4QJT2664)
```
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:81]:
```suggestion
[0.6: Snapshot testing](https://www.youtube.com/watch?v=9Is4QJT2664)
```

File: website/docs/cdktf/examples.html.md [hunk 2]
```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
...
 
-TODO Describe what this is and add a link :-)
+Constributors interested in extending the `cdktf convert` command may be interested in the [convert deep dive](https://www.youtube.com/watch?v=rSn4-Ki5nho). This is not needed to use it.
```
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:83]:
```suggestion
If you're interested in extending the `cdktf convert` command, you may be interested in the [convert deep dive](https://www.youtube.com/watch?v=rSn4-Ki5nho). You do not need to watch this video to use the `convert` command successfully.
```

File: website/docs/cdktf/examples.html.md [hunk 3]
```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
...
+
+## Pocket public codebase
+
+Mozilla Pocket is a widely used application for managing reading lists that's built into the Firefox browser. Like many Mozilla projects, Pocket is open source, and the CDK for Terraform codebase that they use to manage infrastructure for their recommendation API is also public and open source. Pocket's codebase provides a great example of how to lay out a CDK for Terraform project.
```
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:87]:
```suggestion
Mozilla Pocket is a widely used application for managing reading lists that is built into the Firefox browser. Like many Mozilla projects, Pocket is open source, and the CDK for Terraform codebase that Pocket uses to manage infrastructure for the recommendation API is also public and open source. Pocket's codebase provides a great example of how to lay out a CDK for Terraform project.
```

File: website/docs/cdktf/examples.html.md [hunk 4]
```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
...
+
+In order to re-use components, Pocket's codebase is separated out into a [set of reusable modules](https://github.com/Pocket/terraform-modules/tree/main/src/pocket). These are then used from CDK code in the [recommendation-api codebase](https://github.com/Pocket/recommendation-api/tree/main/.aws). The recommended reading order is to:
+
+1. Read [`Pocket/recommendation-api/.aws/src/main.ts`](https://github.com/Pocket/recommendation-api/blob/main/.aws/src/main.ts)
+2. Look at the constructs used that are defined in the [`Pocket/terraform-modules`](https://github.com/Pocket/terraform-modules/tree/main/src/pocket) repository, such as [`PocketALBApplication`](https://github.com/Pocket/terraform-modules/blob/main/src/pocket/PocketALBApplication.ts)
+3. Look at the ["base" constructs](https://github.com/Pocket/terraform-modules/tree/main/src/base), which are are used in the higher-level constructs in the previous step
```
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:93]:
```suggestion
1. Read [`Pocket/recommendation-api/.aws/src/main.ts`](https://github.com/Pocket/recommendation-api/blob/main/.aws/src/main.ts).
2. Look at the constructs used that are defined in the [`Pocket/terraform-modules`](https://github.com/Pocket/terraform-modules/tree/main/src/pocket) repository, such as [`PocketALBApplication`](https://github.com/Pocket/terraform-modules/blob/main/src/pocket/PocketALBApplication.ts).
3. Look at the ["base" constructs](https://github.com/Pocket/terraform-modules/tree/main/src/base), which are are used in the higher-level constructs in the previous step.
```

File: website/docs/cdktf/examples.html.md [hunk 5]
```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
...
+[0.6: snapshot testing](https://www.youtube.com/watch?v=9Is4QJT2664)
 
-TODO Describe what this is and add a link :-)
+Constributors interested in extending the `cdktf convert` command may be interested in the [convert deep dive](https://www.youtube.com/watch?v=rSn4-Ki5nho). This is not needed to use it.
+
+## Pocket public codebase
+
+Mozilla Pocket is a widely used application for managing reading lists that's built into the Firefox browser. Like many Mozilla projects, Pocket is open source, and the CDK for Terraform codebase that they use to manage infrastructure for their recommendation API is also public and open source. Pocket's codebase provides a great example of how to lay out a CDK for Terraform project.
+
+In order to re-use components, Pocket's codebase is separated out into a [set of reusable modules](https://github.com/Pocket/terraform-modules/tree/main/src/pocket). These are then used from CDK code in the [recommendation-api codebase](https://github.com/Pocket/recommendation-api/tree/main/.aws). The recommended reading order is to:
```
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:89]: Did you mean from CDK for Terraform code? Just checking!

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block, but suggesting some nits for you to change. Also - I think we had discussed adding a few sentences of description for each of the examples earlier up on this page so that users know what they are clicking on. I imagine we could pull some context out of the READMEs in those examples. After this PR is merged, would it be helpful for me to pull that context and draft the 1-2 sentence explanations?

File: website/docs/cdktf/examples.html.md (hunk 5 above)
Comment (danieldreier) [reply website/docs/cdktf/examples.html.md:89]: Yes! If you can draft example-summaries that would be a great improvement

**Pull request merged**